python/pfs/drp/stella/measureFiberNorms.py

```diff
@@ -18,11 +18,10 @@
 from lsst.pipe.base.butlerQuantumContext import ButlerQuantumContext
 from lsst.pipe.base.connections import InputQuantizedConnection, OutputQuantizedConnection
 
-from pfs.datamodel import CalibIdentity, PfsConfig
+from pfs.datamodel import CalibIdentity, PfsConfig, PfsArm, PfsFiberNorms
 
 from .combineSpectra import combineSpectraSets
 from .constructSpectralCalibs import setCalibHeader
-from .datamodel import PfsArm, PfsFiberNorms
```

Comment:
I want `pfs.drp.stella.datamodel.PfsArm` instead of `pfs.datamodel.PfsArm`.

Reply:
The question above was really for this comment.